worker/peergrouper/desired.go

```diff
@@ -17,20 +17,20 @@ import (
 	"github.com/juju/juju/network"
 )
 
-// jujuMachineKey is the key for the tag where we save a member's machine id.
-const jujuMachineKey = "juju-machine-id"
```

Comment:
Is it ok to change this `value` without having any compatibility issue?
It is used as the key of Tag in `mongo`.

Reply:
Ah thanks, I didn't intend to change this yet